### DISCUSSION TITLE
Account suspended. waiting for 2 weeks, still got no response.

### DISCUSSION THREAD

**Discussion by osawakoki** [community/community#46947] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello everyone.

Is there anyone who got an account suspended and got unlocked?
I would like to know how long it took to take an account back.

My account got suspended 2 weeks ago and I created a ticket for that immediately.
It's been 2 weeks and received no reply.
I think it is too long...
I got a message that says a templated sentence( We are experiencing high volumes and therefore, you may experience longer than normal wait times.).

I never violate a TOS (creating more than one free account, spreading malware, violating DMCA) and I got no message why my account got suspended.
Plus, I am paying pro plan ($4) a month and Copliot plan ($100) a year.
Copilot is also disables while GitHub charged for that.
I am curious about that and feel frustrated a bit because I am wasting money for no reason.

Most of my code are saved and managed on my GitHub repos.
Therefore, It's devastating that I have no access to these repos.

Can anyone give me a help?
Thank you!

===== info =====
ticket: #1987757
username: osawa-koki
===== === =====

osawa koki

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! 

Tickets are answered in the order they are received and wait time may vary based on our team's resources and the # of tickets ahead of yours. I understand that is frustrating so I really appreciate your patience. Our Support team are the only folks who are able to assist in this matter so I'm going to lock this post to prevent any misinformation or account risks. 

Best of luck!
